src/Backend/Modules/MediaGalleries/Layout/Templates/MediaGalleryAdd.html.twig

```diff
@@ -27,7 +27,7 @@
                   <div class="panel-heading">
                     <p>
                       {{ 'lbl.WidgetView'|trans|ucfirst }}{{ macro.required }}
-                      <abbr class="fa fa-info-circle" data-toggle="tooltip" title="{{ 'msg.WidgetViewsHelp'|trans|ucfirst }}"></abbr><br>
+                      <abbr class="fa fa-info-circle" aria-hidden="true" data-toggle="tooltip" title="{{ 'msg.WidgetViewsHelp'|trans|ucfirst }}"></abbr><br>
```

Comment:
Can you undo this? A better fix is available in https://github.com/forkcms/forkcms/pull/2320